### PR TITLE
Events

### PR DESCRIPTION
**Pull request by kkmanuu**
- I Created a new 'logic' directory to house math logic: To organize our project, I  set up a separate 'logic' directory where   store the 'calculate.js' and 'operate.js' files.

- I Copied 'calculate.js' and 'operate.js': I  copied the required math logic files into the newly created 'logic' directory to ensure a clean project structure.

- I Installed big.js via NPM: To handle precise arithmetic, I  installed the 'big.js' library using NPM, enhancing our calculator's accuracy.

- I Imported 'calculate.js' into 'Calculator.js': To leverage the math logic in our React component, I successfully imported the 'calculate.js' file into the 'Calculator.js' component.

- I Implemented click event handlers:

### CODE REVIEW

File: src/Components/Calculator.js [hunk 1]
```diff
@@ -1,40 +1,75 @@
-import React from 'react';
+/* eslint-disable react/prop-types */
```
Comment (gemmen29) [inline src/Components/Calculator.js]: - [ ] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place.

File: src/Components/Calculator.js [hunk 2]
```diff
@@ -1,40 +1,75 @@
...
+          ['1', 'number'],
+          ['2', 'number'],
+          ['3', 'number'],
+          ['+', 'operation'],
+        ].map(([label, type]) => renderButton(label, type))}
+      </div>
+      <div className="buttons-div">
+        {[
+          ['0', 'number zero'],
+          ['.', 'operation'],
```
Comment (gemmen29) [inline src/Components/Calculator.js]: - [x] Kindly fix the `.` operation button as it's working. When I press the number then I press the `.` operation it displays like that:
image and lead to an error as it's not a valid number.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @kkmanuu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Calculator.js (hunk 2 above)
Comment (kkmanuu) [reply src/Components/Calculator.js]: @gemmen29 I have fix the operation button and it's now working as expected.

File: src/Components/Calculator.js (hunk 1 above)
Comment (kkmanuu) [reply src/Components/Calculator.js]: @gemmen29 I manage to remove the commented eslint-disable everything is working as expected.

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @kkmanuu👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**